src/database/seeders/20191114145912-users.js

```diff
@@ -2,17 +2,22 @@
 
 module.exports = {
   up: (queryInterface, Sequelize) => {
-    return queryInterface.bulkInsert('users', [{
+    return queryInterface.bulkInsert('users', [
+      {
         firstName: 'John',
         lastName: 'Doe',
         email: 'demo@gmail.com',
+        password: '$2b$10$Q9H0gYzslg.oIvqoiWBEfesJoe.HTQT0Ezg5itN/h0a5Mf9gxWWvq',
+        isVerified:'no',
+        signupType:'default',
```

Comment:
I'm afraid that you used wrong value for isVerified. I think it should be true/false